test/application.test.ts

```diff
@@ -104,55 +104,55 @@ describe('Application', function () {
     })
 
     it('returns an authenticated client for installation.created', async () => {
-      const event = {
-        id: '123-456',
+      const event2 = {
         event: 'installation',
+        id: '123-456',
         payload: {
           action: 'created',
-          installation: {id: 1}
+          installation: { id: 1 }
         }
-      }
+      } as any
 
       app.on('installation.created', async context => {
         // no-op
       })
 
-      await app.receive(event)
+      await app.receive(event2)
 
       expect(app.auth).toHaveBeenCalledWith(1, expect.anything())
     })
 
     it('returns an unauthenticated client for installation.deleted', async () => {
-      const event = {
-        id: '123-456',
+      const event2 = {
```

Comment:
Instead of `event2`, I think you can just leave off the `const` declaration and it will re-assign the existing `event` object.